### PR TITLE
Alpha Social

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
Content "Alpha Social trait"

#### Purpose of change
Adds an Alpha social trait that operates similar to Jekyll and Hyde.

#### Describe the solution
Creates a Jekyll and Hyde situation for Alpha mutants that can be triggered by strong emotions or overindulging in drugs.

#### Describe alternatives you've considered
None

#### Testing
Needs a way to set the timer variable.

### CODE REVIEW

File: data/json/mutations/mutations.json [hunk 1]
```diff
@@ -10115,6 +10115,75 @@
...
+    "valid": false,
+    "mixed_effect": true,
+    "flags": [ "ASOCIAL2" ],
+    "category": [ "ALPHA" ],
+    "triggers": [
+      [
+        {
+          "condition": {
+            "and": [
+              { "math": [ "time_since('transform to dark triad', 'unit': 'hours') > 36" ] }
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:10177]:
```suggestion
              { "math": [ "time_since(u_transform_time)", ">", "time('36 h')" ] }
```

File: data/json/mutations/mutations.json [hunk 2]
```diff
@@ -10115,6 +10115,75 @@
...
+    "description": "You are a god among mortals and they are all so very dissapointing.  Why should you even bother to hide your dissapointment in their failures?",
+    "valid": false,
+    "mixed_effect": true,
+    "flags": [ "ASOCIAL2" ],
+    "category": [ "ALPHA" ],
+    "triggers": [
+      [
+        {
+          "condition": {
+            "and": [
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:10176]:
```suggestion
```

**Comment by Kantonine** [comment]
YMMV, but i feel that this could be a worthwile trait for Medical to have as well

**Comment by Maleclypse** [comment]
> YMMV, but i feel that this could be a worthwile trait for Medical to have as well

Medical has a version already with Jekyll and Hyde.   I think I could make it so Alpha in the bad trait isn’t introverted but they are really bad at interactions and I could give medical a trait set that jumps from introverted to extroverted?

**Comment by Vaskritaya** [comment]
> Or even add s god complex trait that requires followers or alpha suffers issues, that one could be a post thresh trait?

Theres that social trait "Sociable" which reduces morale if you havent been around guys in your faction for a while. The vibes of that mutation and its prerequisite don't really fit Alpha, though.

File: data/json/mutations/mutations.json [hunk 3]
```diff
@@ -10115,6 +10115,72 @@
...
+          },
+          "msg_on": { "text": "You give in to your worst instincts.", "rating": "mixed" }
+        }
+      ],
+      [
+        {
+          "condition": {
+            "or": [
+              { "math": [ "u_val('stim')", "<", "-50" ] },
+              { "math": [ "u_val('stim')", "<", "50" ] },
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:10150]:
```suggestion
              { "math": [ "u_val('stim')", "<", "50" ] }
```

File: data/json/mutations/mutations.json [hunk 4]
```diff
@@ -10115,6 +10115,72 @@
...
+    "flags": [ "SOCIAL2" ],
+    "category": [ "ALPHA" ],
+    "social_modifiers": { "intimidate": 5, "persuade": 15, "lie": 10 },
+    "triggers": [
+      [
+        {
+          "condition": {
+            "or": [
+              { "math": [ "u_val('morale')", "<", "-75" ] },
+              { "math": [ "u_val('morale')", "<", "75" ] },
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:10139]:
```suggestion
              { "math": [ "u_val('morale')", "<", "75" ] }
```

File: data/json/mutations/mutations.json [hunk 5]
```diff
@@ -10115,6 +10115,72 @@
...
+      ]
+    ],
+    "transform": { "target": "DARK_TRIAD", "msg_transform": "You give in to your worst instincts.", "active": false, "moves": 10 }
+  },
+  {
+    "type": "mutation",
+    "id": "DARK_TRIAD",
+    "name": { "str": "Dark Triad" },
+    "points": 1,
+    "vitamin_cost": 160,
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:10164]:
```suggestion
```

File: data/json/mutations/mutations.json [hunk 6]
```diff
@@ -10115,6 +10115,71 @@
...
+        }
+      ]
+    ],
+    "transform": { "target": "DARK_TRIAD", "msg_transform": "You give in to your worst instincts.", "active": false, "moves": 10 }
+  },
+  {
+    "type": "mutation",
+    "id": "DARK_TRIAD",
+    "name": { "str": "Dark Triad" },
+    "points": 1,
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:10163]:
```suggestion
```

File: data/json/mutations/mutations.json [hunk 7]
```diff
@@ -10115,6 +10115,70 @@
...
+  },
+  {
+    "type": "mutation",
+    "id": "DARK_TRIAD",
+    "name": { "str": "Dark Triad" },
+    "types": [ "SOCIAL" ],
+    "cancels": [ "ASOCIAL1", "ASOCIAL2", "SOCIAL1", "SOCIAL2" ],
+    "description": "You are a god among mortals and they are all so very dissapointing.  Why should you even bother to hide your dissapointment in their failures?",
+    "valid": false,
+    "mixed_effect": true,
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:10167]:
```suggestion
    "mixed_effect": true,
    "points": -1,
```

File: data/json/mutations/mutations.json [hunk 8]
```diff
@@ -10115,6 +10115,71 @@
...
+      ],
+      [
+        {
+          "condition": {
+            "or": [
+              { "math": [ "u_val('stim')", "<", "-50" ] },
+              { "math": [ "u_val('stim')", "<", "50" ] }
+            ]
+          },
+          "msg_on": { "text": "You give in to your worst instincts.", "rating": "mixed" }
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:10153]:
```suggestion
          }
        }
      ],
      [
        {
          "condition": {
            "or": [
              { "math": [ "u_val('stim')", "<", "-50" ] },
              { "math": [ "u_val('stim')", "<", "50" ] }
            ]
          }
```

File: data/json/mutations/mutations.json [hunk 9]
```diff
@@ -10115,6 +10115,69 @@
...
+    "mixed_effect": true,
+    "points": -1,
+    "flags": [ "ASOCIAL2" ],
+    "category": [ "ALPHA" ],
+    "triggers": [
+      [
+        {
+          "condition": { "math": [ "time_since(u_transform_time)", ">", "time('36 h')" ] }
+          },
+          "msg_on": { "text": "As the well deserved shame of your behaviour finally sets in.", "rating": "good" }
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:10174]:
```suggestion
          }
```

File: data/json/mutations/mutations.json [hunk 10]
```diff
@@ -10115,6 +10115,68 @@
...
+    "mixed_effect": true,
+    "points": -1,
+    "flags": [ "ASOCIAL2" ],
+    "category": [ "ALPHA" ],
+    "triggers": [
+      [
+        {
+          "condition": { "math": [ "time_since(u_transform_time)", ">", "time('36 h')" ] }
+          }
+        }
```
Comment (Maleclypse) [inline data/json/mutations/mutations.json:10174]:
```suggestion
          }
```

**Pull request merged**